website/src/components/ProfileInfo/ProfileInfo.test.jsx

```diff
@@ -12,4 +13,31 @@ describe('ProfileInfo', () => {
   it('Render ProfileInfo without any eror', () => {
     expect(renderer.create(<ProfileInfo name="test full name" email="test@gmail.com" />).toJSON()).toMatchSnapshot();
   });
+
+  /* enzyme testing */
+  const wrapper = shallow(<ProfileInfo name="test full name" email="test@gmail.com" />);
+
+  it('should display name', () => {
+    expect(wrapper.exists('.name')).toBe(true);
+  });
+
+  it('should display email', () => {
+    expect(wrapper.exists('.email')).toBe(true);
```

Comment:
These tests aren't checking for the actual name or email this is checking for a `.email` class. You should instead check if "test full name" appears instead

Reply:
Same comment for other tests

Reply:
I should have clarified You should write better tests to determine if  `test@gmail.com` actually appears in the component instead of checking if the `.email` class appears. Same goes for if `test full name` for the name